poc/pkg/values.yaml

```diff
@@ -90,3 +90,46 @@ AKSCluster:
 keyVault:
   name: aro-rp-aks-kv
   tlsCertificateName: aks-rp-certificate
+
+MISE:
+  container:
+    name: mise
+    image: aropocsvc.azurecr.io/mise-sidecar:dev
+    env:
+      MISE_APPSETTINGS_LOCATION:
+        name: MISE_APPSETTINGS_LOCATION
+        value: "/etc/mise/appsettings.json"
+    ports:
+      containerPort: 5000
+    volumes:
+      name: config
+      mountPath: /etc/mise
+      configMap:
+        name: miseconfig
+        items:
+          key: appsettings.json
+          path: appsettings.json
+  configMap:
+    Instance: https://login.microsoftonline.com
+    ClientId: 704c7dfd-4ed1-4732-95c0-04a44b0895a0
```

Comment:
Is this the client ID of our managed identity?

If so, let's make sure to update the release such that we dynamically set this value. For example, if we re-deploy the managed identity, this value changes.